### ISSUE TITLE
Knockout icons while hovering quick jump search menu.

### ISSUE THREAD

**Issue by the-j0k3r** [StylishThemes/GitHub-Dark#647]
* **Browser**: 66.0.2
* **Operating System**: W10pro 1709
* **Screenshot**:

Im not sure if this is me only because I have a 4K screen running at 2560x1440p or its general.
I suspect this is another upstream bug, but need confirming others also see the issue.

![hover](https://user-images.githubusercontent.com/31389848/41611210-17768944-73e7-11e8-8e90-b7c3665db304.gif)

The gif shows the issue before and after I applied a local fix.

````css
.jump-to-suggestions-path:hover .jump-to-suggestion-name.css-truncate-target {}
````

**Comment by xt0rted** [comment]
I'm seeing this too. It's really subtle, but on hover I can see the icons shift to the left by a single pixel or so.

**Comment by Mottie** [comment]
It looks like all it needs is to set the `max-width`. It'll be available in the next update.

**Comment by the-j0k3r** [comment]
max-width makes the jump to move towards the left.,

**Comment by xt0rted** [comment]
I added a custom user style with the change in b039b25 and it fixes the issue for me with v1.20.49.

**Comment by the-j0k3r** [comment]
It does fixes the knockout the the Jump to is a great deal into the left.
